packages/sth/README.md

```diff
@@ -77,7 +77,11 @@ Copy a content below and save it as `package.json` file:
 Open a terminal run your program on the hub:
 
 ```bash
-si pack /path/to/my/folder -o ~/package.tar.gz # compress the app to a package
+cd /path/to/my/folder
+
+npm install                                    # install dependencies
+
+si seq pack . -o ~/package.tar.gz              # compress the app to a package
```

Comment:
These changes must be implemented in `readme.mtpl` file → packages/sth/src/readme.mtpl